README.md

```diff
@@ -80,7 +80,7 @@ Role Variables
         - `name`: Name to associate with the volume being created; For `file` type volumes include extension if you would like volumes created with one.
         - `file_path`: Where the image of `file` type volumes should be placed; defaults to `libvirt_volume_default_images_path`
         - `device`: `disk` or `cdrom`
-        - `capacity`: volume capacity (can be suffixed with M,G,T or MB,GB,TB, etc) (required when type is `disk`)
+        - `capacity`: volume capacity, can be suffixed with k, M, G, T, P or E when type is `network` or MB,GB,TB, etc when type is `disk` (required when type is `disk`)
```

Comment:
Looks like type is not documented. Could you add it?

Reply:
Please also add auth and source.

Reply:
I'll add some words for auth and source including references to their sub values.

Reply:
This is done in the latest commit/s